lib/mosaic-database/src/main/java/org/eclipse/mosaic/lib/database/spatial/EdgeFinder.java

```diff
@@ -50,14 +49,7 @@ public EdgeFinder(Database database) {
             for (int i = 0; i < con.getNodes().size() - 1; i++) {
                 Node from = con.getNodes().get(i);
                 Node to = con.getNodes().get(i + 1);
-
-                String id;
-                if (database.getImportOrigin().equals(Database.IMPORT_ORIGIN_SUMO)) {
-                    id = con.getId();
-                } else {
-                    id = con.getId() + "_" + from.getId();
-                }
-                items.add(new EdgeWrapper(new Edge(con, id, from, to)));
+                items.add(new EdgeWrapper(new Edge(con, from, to)));
```

Comment:
I don't understand, why we still have to use `Edge`s here? Can't `Connection`s be used here aswell?

Reply:
Connections can contain a list of nodes, hence they can be very long. The map matching algorithm works with rectangles which are shaped from the bounding boxes of the things we put into the search tree. If we would put whole connections, we would retrieve very large and inaccurate bounding boxes, especially for long connections and curves. Instead we break down the connections into separate edges to have much better accuracy.

Reply:
This correlates with a different comment I left. Things now make sense for me.